doc/manual/src/glossary.md

```diff
@@ -3,17 +3,24 @@
 - [derivation]{#gloss-derivation}
 
   A description of a build task. The result of a derivation is a
-  store object. Derivations are typically specified in Nix expressions
+  store object. Derivations created by Nix expressions are specified
   using the [`derivation` primitive](./language/derivations.md). These are
   translated into low-level *store derivations* (implicitly by
   `nix-env` and `nix-build`, or explicitly by `nix-instantiate`).
 
+  Package definitions, such as in [nixpkgs], are written using one or
+  more derivations.
+
   [derivation]: #gloss-derivation
+  [nixpkgs]: https://github.com/NixOS/nixpkgs
 
 - [store derivation]{#gloss-store-derivation}
 
   A [derivation] represented as a `.drv` file in the [store].
-  It has a [store path], like any [store object].
+  It has a [store path], like any [store object]. It is the
+  [instantiated][instantiate] form of a derivation. These are built
+  into *outputs* (implicitly by `nix-env` and `nix-build`, or explicitly
```

Comment:
```suggestion
  into *outputs* (implicitly by `nix-build`, or explicitly
```
We should refrain from mentioning `nix-env` as it's outdated and @fricklerhandwerk and the docs team have put in some good effort to remove it from official materials.